core/src/test/scala/io/projectglow/transformers/normalizevariants/NormalizeVariantsTransformerSuite.scala

```diff
@@ -199,12 +204,50 @@ class NormalizeVariantsTransformerSuite extends GlowBaseTest with GlowLogging {
   }
 
   test("normalization transform no-reference") {
-    try {
-      testNormalizedvsExpected(vtTestVcfMultiAllelic, vtTestVcfMultiAllelic, None)
-    } catch {
-      case _: IllegalArgumentException => succeed
-      case _: Throwable => fail()
-    }
+    assertThrows[IllegalArgumentException](
+      testNormalizedvsExpected(
+        vtTestVcfMultiAllelic,
+        vtTestVcfMultiAllelic,
+        None
+      )
+    )
+  }
+
+  test("Invalid reference path ") {
+    assertThrows[SparkException](
```

Comment:
For these tests, can we check the exception's error message to make sure the user sees something reasonable?

Reply:
I have checked. They see proper messages from htsjdk.

Reply:
Could we catch the error messages with `intercept` to verify in the unit tests?

Reply:
Done